src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,17 @@
+import { useState } from "react";
+import "./checkbox.scss";
+
+export default function Checkbox() {
+  const [checked, setChecked] = useState(false);
+
+  function toggle() {
+    setChecked(!checked);
+  }
+
+  const checkedClass = checked ? "checkbox--checked" : "";

```

Comment:
Let's use the library `classnames` as always, like so:

```suggestion
  const checkedClass = cn('checkbox', { 'checkbox--checked': checked });
```
Doing that, you're gonna not need `containerClass` down bellow